docs/pages/versions/unversioned/introduction/testing-with-jest.md

```diff
@@ -0,0 +1,173 @@
+---
+title: Testing with Jest
+---
+
+Adding Jest to your Expo app has gotten pretty simple, to help with this process, we've even created a [Jest preset](https://github.com/expo/expo/tree/master/packages/jest-expo) for you. Let's get started.
+
+## Installation
+
+First we need to install jest-expo: `yarn add jest-expo --dev` **or** `npm i jest-expo --save-dev`
+
+Then we need to add/update `package.json` to include:
+
+```js
+"scripts": {
+  ...
+  "test": "jest"
+},
+"jest": {
+  "preset": "jest-expo"
+}
+```
+
+Now let's add [enzyme](https://airbnb.io/enzyme/), [enzyme adapter for react < 16](https://airbnb.io/enzyme/#upgrading-from-enzyme-2x-or-react--16) and [react-dom](https://github.com/facebook/react/tree/master/packages/react-dom):
```

Comment:
I think we should use `react-test-renderer` or `[react-native-testing-library](https://github.com/callstack/react-native-testing-library)` (which is build on top of `react-test-renderer`) rather than enzyme. If you create a tabs template project we use `react-test-renderer` directly, so maybe we can just use that here and let people know that `react-native-testing-library` exists and they may find it helpful

Reply:
Thanks! Sorry about that, good to know.